packages/bitcore-wallet-service/src/lib/expressapp.ts

```diff
@@ -1745,6 +1745,19 @@ export class ExpressApp {
       });
     });
 
+    router.post('/v1/service/sardine/getSupportedTokens', (req, res) => {
+      getServerWithAuth(req, res, server => {
```

Comment:
I would not use getServerWithAuth to fetch supported tokens/currencies. You can do it directly like this example:

router.post('/v1/service/ramp/assets', async (req, res) => {
      let server, response;
      try {
        server = getServer(req, res);
        response = await server.rampGetAssets(req);
        return res.json(response);
      } catch (err) {
        return returnError(err ?? 'unknown', res, req);
      }
    });